includes/specials/SpecialListfiles.php

```diff
@@ -233,9 +243,11 @@ function formatValue( $field, $value ) {
 				}
 			case 'img_user_text':
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this "switch/case" number of lines from 12 to at most 10, for example by extracting code into function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)